### ISSUE TITLE
Missing colon separator before port?

### ISSUE THREAD

**Issue by frischnetz** [Mark-H/CookieList#8]
On one https site, clicking on "Add to your CookieList" without any further options, the link redirected to

`https://www.example.com443/pagename/info.html` instead of 
`https://www.example.com/pagename/info.html`

This looks like a missing colon in front of port number (443). To check, I changed line 166 in `cookielist.class.php` to

`return $protocol.'://'.$_SERVER['SERVER_NAME'].':'.$port.$params;`

The redirect now works. Not sure if this is the correct solution, though.

**Comment by Mark-H** [comment]
That shouldn't be needed, the colon is already added to $port for non-80 ports two lines before:

https://github.com/Mark-H/CookieList/blob/master/core/components/cookielist/model/cookielist.class.php#L164

```
$port = $_SERVER['SERVER_PORT'] == '80' ? '' : ':'.$_SERVER['SERVER_PORT'];
```

... unless that fix was never released. 🙈

**Comment by frischnetz** [comment]
Haha, ok, I see. Looking forward to the fixed version! This package is a neat simple solution for a wish of my client.

**Comment by Mark-H** [comment]
Fixed in 1.1
